core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.ts

```diff
@@ -71,9 +88,10 @@ export class BaseUser {
         args,
       })
       .then(async browser => {
-        ConsoleReporter.trackConsoleMessagesInBrowser(browser);
+        this.startTimeInMilliseconds = Date.now();
         this.browserObject = browser;
         this.page = await browser.newPage();
+        ConsoleReporter.trackConsoleMessagesInBrowser(browser);
```

Comment:
This might actually break the console reporter since it relies on the targetcreated event to track a new page but the new page is being called beforehand.